### PR TITLE
[IMP] email comms: feedback loop edit

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3714944&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@StraubCreative 

This is a small change, brought to my attention by a consultant in BE. The original system parameters given to me for this change were incorrect.

Confirmed with mail expert.

CC: @lma-odoo

File: content/applications/general/email_communication/email_servers.rst [hunk 1]
```diff
@@ -284,22 +284,22 @@ in the *System Parameters*. To override the following defaults they need to be a
...
 Change the value of these parameters, as needed.
 
 When an email is received in the Odoo database on the catchall email address or on any alias, Odoo
 looks at the mail received for the given period of time defined in the system parameter
-`mail.incoming.limit.period`. If the received email was sent to an alias then Odoo will reference
-the `mail.incoming.limit.alias` system parameter and determine the value as the number of records
-this alias is allowed to create in the given period of time (value of `mail.incoming.limit.period`).
+`mail.gateway.loop.minutes`. If the received email was sent to an alias then Odoo will reference
+the `mail.gateway.loop.threshold` system parameter and determine the value as the number of records
+this alias is allowed to create in the given period of time (value of `mail.gateway.loop.minutes`).
```
Comment (StraubCreative) [inline content/applications/general/email_communication/email_servers.rst:298]: early line breaks
```suggestion
`mail.gateway.loop.minutes`. If the received email was sent to an alias then Odoo will reference the
`mail.gateway.loop.threshold` system parameter and determine the value as the number of records this
alias is allowed to create in the given period of time (value of `mail.gateway.loop.minutes`).
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]

**Comment by tiku-odoo** [comment]
@StraubCreative 

This PR is now ready for merge. 

Thanks. Tim :+1:

**Review by StraubCreative — APPROVED** [review]
Thanks @tiku-odoo 👍 
...
@robodoo r+

**Pull request closed without merging**